test/build/base-url/base-url.test.js

```diff
@@ -1,15 +1,20 @@
-const fs = require('fs');
 const path = require('path');
 const cheerio = require('cheerio');
+const {setupBuildTest, readFiles} = require('../../test-utils');
 
 const cwd = path.join(__dirname, 'build');
 
-const html = fs.readFileSync(path.join(cwd, 'index.html'), 'utf8');
-const indexJS = fs.readFileSync(path.join(cwd, 'index.js'), 'utf-8');
-const distJS = fs.readFileSync(path.join(cwd, '_dist_', 'index.js'), 'utf-8');
-const $ = cheerio.load(html);
+let files = {};
+let $;
 
 describe('buildOptions.baseUrl', () => {
```

Comment:
also, top-level `describe` name should match the file name so that we keep the current behavior of `yarn test base-url`. Right now, we have the extra overhead of keeping track of two different names

Reply:
I’m game for renaming the `describe()` names to be clearer in purpose, but I’m against renaming them to match filenames. No matter what they’re named, you still have to specify part of the filename with `yarn test base-url`. But they’re an opportunity to explain a test’s focus in more clear language, and not just repeat the folder name (Jest shows you both). But to that end, I’m not saying that they are human-readable clear language now; that’s just where I’d like them to be.

Reply:
Whoops, looks like I misunderstood. `yarn test base-url` seems to work in the existing `main` branch. I could have sworn I tried this and it didn't work. nvm!